docs/content/v0.20/example-wecs.md

```diff
@@ -4,29 +4,23 @@ The following steps show how to create new clusters and
 register them with the hub as descibed in the
 [official open cluster management docs](https://open-cluster-management.io/getting-started/installation/start-the-control-plane/).
 
-1. Run the following set of commands for creating a new kind cluster with name "cluster1" and registering it with the
+1. Run the following set of commands for creating two kind clusters with name `cluster1` and `cluster2` and registering it with the
```

Comment:
"name" is singluar but we are talking plural here.
I would s/with name/named/

Reply:
ok, done